pkgs/development/python-modules/scspell/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  nix-update-script,
+  pyxdg,
+  setuptools,
+  writableTmpDirAsHomeHook,
+}:
+
+buildPythonPackage rec {
+  pname = "scspell";
+  version = "2.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "myint";
+    repo = "scspell";
+    tag = "v${version}";
+    hash = "sha256-XiUdz+uHOJlqo+TWd1V/PvzkGJ2kPXzJJSe5Smfdgec=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    pyxdg
+  ];
+
+  nativeCheckInputs = [ writableTmpDirAsHomeHook ];
+
+  passthru.tests = {
+    version = {
+      command = [
+        "scspell"
+        "--version"
+      ];
+    };
+    help = {
+      command = [
+        "scspell"
+        "--help"
+      ];
+    };
+    basic-check = {
+      command = ''
+        export HOME=$TMPDIR
+        cat > test_file.py << 'EOF'
+        def hello_world():
+            print("Hello, world!")
+        EOF
+        scspell test_file.py
```

Comment:
Similar here; as `passthru` is not run when building the package, this doesn't really help you.
As I've seen you ignore `scspell` in many of your other PRs, I imagine the tests of the other PRs passing provides a better test than this `passthru` block.